server/src/inmantals/server.py

```diff
@@ -682,7 +697,11 @@ async def textDocument_hover(self, textDocument, position):
         data = list(range)[0].data
         docstring = textwrap.dedent(data.docstring.strip("\n")) if data.docstring else ""
         docstring = docstring.replace(" ", "&nbsp;").replace("\n", "\n\n").strip()
-        definition = self.get_definition(data).strip()
+        try:
```

Comment:
Fixes a bug when hovering over the `Entity` keyword e.g. in 
```
implementation none for Entity:
    """
        An empty implementation that can be used as a safe default.
    """
end
```
As I understand it this keyword is defined in a dummy file called "internal" that isn't meant to be navigated to ?